rpcs3/Emu/System.h

```diff
@@ -280,6 +280,7 @@ class Emulator final
 
 	void SetUsr(const std::string& user);
 
+	static std::string GetBackgroundPicturePath(const std::string& sfo_dir, const std::string& title_id);
```

Comment:
I don't think this is the correct function.
This one gets the background for the shader dialogs

Reply:
Fixed.